Tasks/GitHubPRComment/githubComment/index.ts

```diff
@@ -3,8 +3,8 @@ import gitClient = require('@octokit/rest');
 import path = require('path');
 import fs = require('fs');
 
-//We use HTML comments to hide the flagComment in the comment.
-const flagComment = "<!-- Commented using GitHubPRComment. -->";
+// We use HTML comments to hide the flagComment in the comment.
+const flagComment = "<!-- Commented using GitHubPRComment. -->"; // DO NOT DELETE THESE COMMENT
```

Comment:
The comment `DO NOT DELETE THESE COMMENT` i meant to be inside the flag comment, its just a clarification for the github comment, for the person that reads it